pkg/config/mesh/mesh.go

```diff
@@ -43,7 +43,7 @@ func DefaultProxyConfig() meshconfig.ProxyConfig {
 		ServiceCluster:           constants.ServiceClusterName,
 		DrainDuration:            types.DurationProto(45 * time.Second),
 		ParentShutdownDuration:   types.DurationProto(60 * time.Second),
-		TerminationDrainDuration: types.DurationProto(5 * time.Second),
+		TerminationDrainDuration: types.DurationProto(45 * time.Second),
```

Comment:
Discussed a bit more in the issue, but we need to make sure we don't make this a breaking API change